apps/explorer/src/pages/id-page/PageContent.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetObject } from '@mysten/core';
+import { Banner } from '~/ui/Banner';
+import { Divider } from '~/ui/Divider';
+import { FieldsContent } from '~/pages/object-result/views/TokenView';
+import { TabHeader } from '~/ui/Tabs';
+import { ErrorBoundary } from '~/components/error-boundary/ErrorBoundary';
+import { TransactionsForAddressTable } from '~/components/transactions/TransactionsForAddress';
+import TransactionBlocksForAddress from '~/components/TransactionBlocksForAddress';
+import { useBreakpoint } from '~/hooks/useBreakpoint';
+import { OwnedCoins } from '~/components/OwnedCoins';
+import { OwnedObjects } from '~/components/OwnedObjects';
+import { LOCAL_STORAGE_SPLIT_PANE_KEYS, SplitPanes } from '~/ui/SplitPanes';
+import { useSuiClient } from '@mysten/dapp-kit';
+import { useQuery } from '@tanstack/react-query';
+
+const LEFT_RIGHT_PANEL_MIN_SIZE = 30;
+
+function OwnedObjectsSection({ address }: { address: string }) {
```

Comment:
I'd break all these sections out into their own files just to avoid having this mega file.